frontend/.gitignore

```diff
@@ -2,6 +2,7 @@
 /node_modules
 /.pnp
 .pnp.js
+/devlink
```

Comment:
I added `/devlink` to the `.gitignore`, but since it wasn't initially ignored, Git is still tracking it. 
I'm unsure whether we should stop tracking it entirely.
 Should we consider removing it from Git's tracking?